test/rest_api/test_storage_service.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021-present ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import sys
+import requests
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table
+
+def test_storage_service_auto_compaction(cql, rest_api, test_keyspace):
+    with new_test_table(cql, test_keyspace, "a int, PRIMARY KEY (a)") as t:
+        resp = rest_api.send("DELETE", f"storage_service/auto_compaction/{test_keyspace}")
+        resp.raise_for_status()
+
+        resp = rest_api.send("POST", f"storage_service/auto_compaction/{test_keyspace}")
+        resp.raise_for_status()
```

Comment:
Both points are correct.
But since the default auto-compaction state might theoretically change in the future maybe it is a good idea to create a temporary keyspace for the test (for each case since it's not a fixture anymore)

Reply:
Yes, I think it will be better. I'm tempted to commit this as-is and let you send a followup patch for this if you want.